### ISSUE TITLE
Drush 8 + backdrop commands: `drush en` fails on Drupal 7 site

### ISSUE THREAD

**Issue by jenlampton** [backdrop-contrib/drush#64]
Almost all my backdrop drush commands are working perfectly, but I still spend a lot of time on Drupal7 sites too.  Recently the `drush en devel -y` command, when executed within a Drupal 7 site, will return the following error:

```
Error devel does not exist in your Backdrop installation.
Try downloading devel first with the command: drush dl devel
```

**Comment by jenlampton** [comment]
I tried the solution recommended by @quicksketch in https://github.com/backdrop-contrib/drush/issues/46#issuecomment-235455170, but unfortunately that prevents `drush en` from doing anything at all on Drupal 7 sites.

**Comment by serundeputy** [comment]
@jenlampton is this still relevant? maybe was fixed by #7?

let's close if this is moot now.

**Comment by jenlampton** [comment]
It worked with my testing, closing issue.
